meta/meta.go

```diff
@@ -779,8 +779,8 @@ func (m *Meta) GetLastHistoryDDLJobsIterator() (*LastJobIterator, error) {
 	}, nil
 }
 
-// GetLastJobs gets last several jobs.
-func (i *LastJobIterator) GetLastJobs(num int, jobs []*model.Job) ([]*model.Job, error) {
+// GetNextJobs gets next several jobs.
+func (i *LastJobIterator) GetNextJobs(num int, jobs []*model.Job) ([]*model.Job, error) {
```

Comment:
Why change this function name? I think the last job is better.

Reply:
Em..., last mostly means the end, ok, it also can stand the last job in the LastJobIterator.
Changed back.